backend/handlers/handlers.go

```diff
@@ -216,6 +216,7 @@ func NewHandlers(
 	getAPIRouterNoError(apiRouter)("/testing", handlers.getTesting).Methods("GET")
 	getAPIRouterNoError(apiRouter)("/account-add", handlers.postAddAccount).Methods("POST")
 	getAPIRouterNoError(apiRouter)("/keystores", handlers.getKeystores).Methods("GET")
+	getAPIRouterNoError(apiRouter)("/keystore-name", handlers.getKeystoreName).Methods("POST")
```

Comment:
Why POST method? We usually use POST for calls that change data or that actively do something. In this case, since you are getting data instead of modifying it you should use a GET